lua/modules/controls.lua

```diff
@@ -207,7 +207,7 @@ local _isActive = function()
 	return _state.on and menu:IsActive() == false
 end
 
-local _diffuseLongPressTimer = function()
+local _defuseLongPressTimer = function()
```

Comment:
The function name referred as something that transmits the long press timer rather than cancelling it, I hope I understood well